scripts/composer/ScriptHandler.php

```diff
@@ -40,11 +40,9 @@ public static function createRequiredFiles(Event $event) {
       $fs->copy($drupalRoot . '/sites/default/default.settings.php', $drupalRoot . '/sites/default/settings.php');
       require_once $drupalRoot . '/core/includes/bootstrap.inc';
       require_once $drupalRoot . '/core/includes/install.inc';
-      $settings['config_directories'] = [
-        CONFIG_SYNC_DIRECTORY => (object) [
-          'value' => Path::makeRelative($drupalFinder->getComposerRoot() . '/config/sync', $drupalRoot),
-          'required' => TRUE,
-        ],
+      $settings['config_directories'] = (object) [
```

Comment:
I think this should be `$settings['config_sync_directory']` as per https://www.drupal.org/node/3018145 but I'm not sure (https://github.com/UN-OCHA/pocam8-site/pull/78#discussion_r677593127)

Reply:
Nice catch, fixed